src/codegen/compile.ml

```diff
@@ -2497,7 +2497,7 @@ module BigNumLibtommath : BigNumType = struct
         then []
         else
           let (a, b) = Big_int.quomod_big_int n twoto28 in
```

Comment:
Curious why it's 28 bit here?

Reply:
@nomeata is this some MP thing?

Reply:
Yes, I think the internal representation of libtommath bigints uses 28 bit limbs